pages/learn/manage/organizations.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+An **organization** in balenaCloud is a group of members, teams, and applications with control over accessiblity and visiblity. Organizations allow granular access control at the application level through multiple roles that can be assigned to individual nembers or teams. This in turns helps to improve the visibility of who has access to which applications making collaboration easier. 
+
+Organizations are available for use in all balenaCloud accounts, and multiple organizations can be created by a single user. Organizations blend in with existing features and workflows improving upon what's already there.
```

Comment:
A sentence like _"Organizations blend in with existing features and workflows improving upon what's already there"_ is not really helpful to readers of this document in my opinion, and could be deleted.

_"multiple organizations can be created by a single user"_ does not make it clear whether such a user is using one or more balenaCloud user accounts. I think the following is a more useful fact to be documented (as always, assuming it is correct...).

```
A balenaCloud account may contain one or more organizations, but a balena application
may be added to a single organization.
```